src/ResultsActivity/elements/PayOptions/PayOptions.js

```diff
@@ -46,35 +46,46 @@ function PayOptions({ offenderDetails, updateOffenderAttendance, date, openModal
       />
     )
 
-  const allowUpdate = () => {
-    if (selectedOption === 'other' && isWithinLastYear(date)) return true
-    return false
+  const allowUpdate = selectedOption === 'other'
+
+  const showRadioButton = !locked && offenderNo && eventId && isWithinLastWeek(date)
+
+  const payMessage = () => {
+    if (paid && locked) return 'Paid'
+    if (!paid && locked) return 'Not paid'
+    if (!paid && !other && !isWithinLastWeek(date)) return 'Not Recorded'
+    return null
+  }
+
+  const otherMessage = () => {
```

Comment:
Don't think we need this now? Can just use `other &&` instead of `otherMessage() &&` and stick View/Update within `<OtherMessage />`